### ISSUE TITLE
You do not have sufficient permissions to access this page...

### ISSUE THREAD

**Issue by ntkntk2** [engintron/engintron#537]
Hi 
I have just installed a Engintron on fresh WHM/Cpanel CentOS Linux release 7.3.1611 (Core)
cPanel version: 64.0 (build 19).
When I try to open in WHM interface the management page for Engintron I see white page " You do not have sufficient permissions to access this page..." 
When I try to open a site from my server it gives me "502 Bad Gateway". 
So I completely stuck after the installation. 
Can you give me advice where too look for a solution for this error - the one with the permisssions?
I have uninstallled it and installed it again - but the problem is the same.
regards
Nikolay Kabaivanov

**Comment by spirogg** [comment]
edit your custom rules for nginx
at or around line 25

#set $PROXY_DOMAIN_OR_IP "xxx.xxx.xxx.xxx"; # Use your cPanel's shared IP address here
just remove the first # and add your shared IP Address from your server
this also works without doing the other steps below , i just noticed this...

then save the file and restart and purge cache and temp files
all done from cpanel enginetron plugin


(if the above steps don't work try the below)
---------------------------------------------------------------
add your domain and your ip

find this around line 73

#if ($host ~ "example.com") {
#set $PROXY_DOMAIN_OR_IP "XXX.XXX.XXX.XXX";
#}

then add this below that without the #

if ($host ~ "yourdomain-goes-here.com") {
set $PROXY_DOMAIN_OR_IP "your.shared.ip.goes.here";
}
then save the file and restart and purge cache and temp files

thats the way i got mine to work i was getting 502 error and this resolved my issue. if you have more than 1 domain keep adding the if statement with different domain etc. Im sure there is an easier way but this worked for me now

**Comment by ntkntk2** [comment]
Hi, I have found what caused my problems. 
I have used web interface of inmotion hosting to log me as a root in WHM. This is causing the problem with lack of rights. When I logon directly at the server address http://IP.add.re.ss/whm the problem with the insufficient right is gone.
The second problem with "502 Bad Gateway" is caused by DNS. When I tested my site I have used my local hosts file to trick my browser to open it from the new server, but nginx is using the google's resolvers and this causing is to open real site at port 8080. The real site was on the production server and I have not pointed it's DNS records to this Cpanel server. 
regards
Nikolay Kabaivanov
